packages/edit-navigation/src/index.js

```diff
@@ -31,6 +32,43 @@ function disableInsertingNonNavigationBlocks( settings, name ) {
 	return settings;
 }
 
+function removeNavigationBlockSettingsUnsupportedFeatures( settings, name ) {
+	if ( name !== 'core/navigation' ) {
+		return settings;
+	}
+
+	return {
+		...settings,
+		supports: {
+			...omit( settings.supports, [
+				'anchor',
+				'customClassName',
+				'__experimentalColor',
+				'__experimentalFontSize',
+			] ),
+			customClassName: false,
+		},
+	};
+}
```

Comment:
I would use an allowlist, not denylist here. With an allowlist, worst case scenario is not offering a new feature on this screen. With a denylist, worst case scenario is breaking the screen with unsupported feature that was added without considering this editor.

Reply:
I'm not sure that would be possible as block `supports` settings are an already defined public API.

> With a denylist, worst case scenario is breaking the screen with unsupported feature that was added without considering this editor.

I'm not sure it'd break anything, it'd be a setting that would never have its value saved.

Reply:
I suppose the option would be to use a filter to add them to the post/site editor at registration, though I personally feel that blocks in the block library shouldn't require any extra configuration, they should work as intended out of the box.